src/main/scala/lms/thirdparty/c_lib_utils.scala

```diff
@@ -27,8 +27,8 @@ trait CCodeGenCMacro extends ExtendedCCodeGen {
 }
 
 trait LibStruct { b : Base =>
-  def newStruct[T: Manifest]: Rep[T] = {
-    Wrap[T](Adapter.g.reflectUnsafe("lib-struct", Backend.Const(manifest[T])))
+  def newStruct[T: Manifest](type_name: String): Rep[T] = {
```

Comment:
```suggestion
  def newStruct[T: Manifest](typeName: String): Rep[T] = {
```

https://docs.scala-lang.org/style/naming-conventions.html